themes/openy_themes/openy_carnation/src/scss/modules/_global-search.scss

```diff
@@ -44,6 +77,57 @@
 
     .views-field-title {
       font-weight: bold;
+      margin-bottom: 1rem;
+      a {
+        font-size: 26px;
+        color: #0089d0;
+      }
+    }
+  }
+  nav.pager {
+    text-align: center;
+
+    a {
+      border-radius: 8px;
+      background-color: #f8f8f8;
+      border: solid 1px #f8f8f8;
+      color: #c6168d;
+      width: 50px;
+      height: 50px;
+      padding: 0;
+      line-height: 50px;
+      font-family: "Cachet", Arial, sans-serif;
+      font-size: 20px;
+      font-weight: 800;
+      display: inline-block;
+    }
+    .is-active a {
+      color: #363435;
+      font-size: 20px;
+      font-weight: bold;
+    }
+
+    .pager__item--previous a,
+    .pager__item--next a {
+      background: transparent;
+      border-radius: 8px;
+      border: solid 1px #e7e7e7;
+      color: #0089d0;
+      font-weight: 400;
+      span {
+        display: none;
+      }
+    }
+
+    .pager__item--next a:before {
+      content: "\f054";
+      font-family: 'Font Awesome 5 Free';
```

Comment:
same here, fonts.scss
Please, use the best OOO practices and inheritance from SASS as much as we can, please. No rush here, we need to think about our future as much as we could.

Reply:
Fixed